core-bundle/src/Resources/contao/templates/backend/be_widget_pwchange.html5

```diff
@@ -0,0 +1,9 @@
+
+<div class="widget">
+  <label for="password"><?= $this->label ?></label>
+  <input type="password" name="password" id="password" class="tl_text" value="" autocomplete="new-password" placeholder="<?= $this->label ?>" required>
+</div>
+<div class="widget">
+  <label for="confirm"><?= Contao\System::getContainer()->get('translator')->trans('MSC.confirm.0', [], 'contao_default') ?></label>
+  <input type="password" name="password_confirm" id="confirm" class="tl_text" value="" autocomplete="new-password" placeholder="<?= Contao\System::getContainer()->get('translator')->trans('MSC.confirm.0', [], 'contao_default') ?>" required>
```

Comment:
Why not use `$this->trans()` here?

Reply:
What do you mean by "switch back to $GLOBALS"? The original code is this:

`<label for="confirm"><?= $this->confirm ?></label>`

It seems we have to add something like this:

```
$widget->password = $GLOBALS['TL_LANG']['MSC']['password'][0];
$widget->confirm = $GLOBALS['TL_LANG']['MSC']['confirm'][0];
```

Reply:
> It seems we have to add something like this:

I do not think we should set translation strings for a widget's template this way. But if you prefer that over `<?= $GLOBALS['TL_LANG']… ?>` in the template I will change it.

Reply:
Either we add `$this->trans()` for widgets or we set the translated string as template variable (like we do everywhere else). @ausi What would you prefer?

Reply:
I think I’d set the template variables as you outlined in https://github.com/contao/contao/pull/5415/files#r1004633603

Side note: we also use `$GLOBALS['TL_LANG']` directly in some templates (like `be_welcome` for example).

Reply:
Changed in 777a030d6b